src/platform/silabs/wifi/WifiInterfaceAbstraction.cpp

```diff
@@ -172,11 +175,11 @@ void wfx_ip_changed_notify(int got_ip)
  *      with AP continously after a certain time interval.
  * @param[in]  retryAttempt
  * @return None
- ********************************************************************************************/
+ *************************************************************************************/
 void wfx_retry_connection(uint16_t retryAttempt)
 {
     // During commissioning, we retry to join the network MAX_JOIN_RETRIES_COUNT
-    if (BaseApplication::sAppDelegate.isCommissioningInProgress())
+    if (/*BaseApplication::sAppDelegate.isCommissioningInProgress()*/ true)
```

Comment:
The retry logic is modified and which may leads to AP and DUT power cycle issues. Need to test all scenarios like retry before joining the AP, retry after commissioning, and all power cycle scenarios for both the AP and DUT. 
If everything works as expected, we can remove the else code?

Reply:
Good catch! I'll fix this in a follow up to avoid mixing a cherry-pick and an sl-up PR